sageresearch-app-sdk/src/main/java/org/sagebionetworks/research/sageresearch/dao/room/EntityTypeConverters.kt

```diff
@@ -62,7 +68,7 @@ import java.util.UUID
  */
 class EntityTypeConverters {
 
-    private val bridgeGson = GsonBuilder()
```

Comment:
This probably should extend RestUtils.GSON. Could you add a TODO?

RestUtils should expose the GsonBuilder so we can add to it